nova/scheduler/weights/resize_same_shard.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2022 SAP
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+Prefer resize to same shard weigher. Resizing an instance gives the current
+aggregate a higher priority.
```

Comment:
better: `current shard aggregate`, because we don't support just any aggregate

Reply:
fixed